packages/contractkit/src/utils/phone-number-lookup/phone-number-lookup.ts

```diff
@@ -97,27 +98,39 @@ export async function postToPhoneNumPrivacyService<ResponseType>(
   }
 
   const { pgpnpUrl } = context
-  const res = await fetch(pgpnpUrl + endpoint, {
-    method: 'POST',
-    headers: {
-      Accept: 'application/json',
-      'Content-Type': 'application/json',
-      Authorization: authHeader,
-    },
-    body: bodyString,
-  })
-
-  if (!res.ok) {
+
+  const tries = 3
+  let lastError
+  for (let i = 1; i <= tries; i++) {
```

Comment:
Why do we need retry logic? are we seeing intermittent failures?

Reply:
This is more preemptive since network calls commonly fail. We couldn't add it before without potentially consuming the user's quota. Now with Alec's change, we can safely retry with the same timestamp without worrying about quota consumption.

Reply:
I'm not sure polluting the code with retry logic is necessary here, we would usually reserve it for cases where the system is known to be flaky which I think isn't the case for ODIS. As far as general network flakiness goes, TCP should handle things mostly for us anyway. 
My concern is partly just to keep the code simple as possible and partly to avoid noise in partner's signers until they get the update.